Makefile

```diff
@@ -111,6 +111,14 @@ fmt_license: addlicense ## Ensure the license header is set on all files
 	$(ADDLICENSE) -v -f license_header.txt $$(find . -not -path '*/\.*' -name '*.go')
 	$(ADDLICENSE) -v -f license_header.txt $$(find . -name '*ockerfile')
 
+.PHONY: gosec
+gosec: ## run the gosec scanner for non-test files in this repo
+  	ifeq ($(shell command -v gosec 2> /dev/null),)
```

Comment:
To make sure we all use the same version (and do not use any `gosec` installed globally on the system), could you download it locally (to `$(LOCALBIN)`) if necessary? See how we are handling other tool binaries like [`golangci-lint`](https://github.com/janus-idp/operator/blob/main/Makefile#L235-L238) or [`controller-gen`](https://github.com/janus-idp/operator/blob/81e7506b3c86fb7b6a374a28649e27df6ba5a9fb/Makefile#L225-L228).

Reply:
FYI, I locked it down to a specific version because pulling the latest has broken us in the past especially in areas with go version compatibility.  We should update regularly when we can (and rely on renovate to help us here once that's enabled)

Reply:
I totally agree, this is a better approach